packages/api/db/migration/20231219050204_create_release_badge_table.js

```diff
@@ -0,0 +1,27 @@
+/*
+ *  Copyright 2023 LiteFarm.org
+ *  This file is part of LiteFarm.
+ *
+ *  LiteFarm is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  LiteFarm is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details, see <https://www.gnu.org/licenses/>.
+ */
+
+export const up = async function (knex) {
+  await knex.schema.createTable('release_badge', (table) => {
```

Comment:
Not against a new table per se -- but just curious what the value is to adding a new table versus keeping the same functionality on 
a) a new column in spotlights (since its kind of a version spotlight)
or
b) a new column on user

Reply:
No particular reason, other than it was simpler to write the logic I wanted from scratch rather than complicate the Spotlight saga, controllers, and model with new behaviour (e.g. that one is boolean based but I needed the string comparison here, and of course the frontend handling is quite different). I never thought about the `user` table at all actually, but I think the thematic coupling is a lot tighter to Spotlight.

Having finished it, I think it IS easier to read and reason about this way, but I'm curious what @antsgar thinks.

Reply:
I originally thought this was going to be a new column in the spotlight table too, but since this isn't really a spotlight I like the idea of keeping it isolated to make the feature easier to understand -- otherwise you might think there should be a spotlight related to this when there isn't. As for the user table, if it was just the app version column we could have gone down that path, but if there's any usefulness to having an updated at / created at specifically for the app version entry I prefer to keep all of it in its own table rather than bloat the user table with 3 new columns